_activities/circuits.md

```diff
@@ -4,22 +4,19 @@ layout: activity
 # Content
 #
 title: "Circuits and Electricity"
-title_fr: "Électricité et Circuits"
```

Comment:
This file should not have been edited (it is being reverted to a previous version here.)